bin/out

```diff
@@ -44,6 +44,39 @@ start_job() {
     # TODO make sure job is started
 }
 
+delete_resource() {
+    RESOURCE_TYPE=$1
+    RESOURCE=$2
+
+    [ -n "$RESOURCE_TYPE" ] || exit 1
+    [ -n "$RESOURCE" ] || exit 1
+
+    $KUBECTL delete "$RESOURCE_TYPE" "$RESOURCE" || true
+}
+
+create_resource() {
+    RESOURCE_FILE=$1
+    IMG=$2
+ 
+    [ -n "$RESOURCE_FILE" ] || exit 1
+    [ -n "$IMG" ] || exit 1
+
+    
+    sed -i "s|image:.*|image: $IMG|g" $RESOURCE_FILE
+
+    $KUBECTL create -f "$RESOURCE_FILE"
+}
+
+rise_from_the_ashes() {
+    RESOURCE_TYPE=$1
+    RESOURCE=$2
+    RESOURCE_FILE=$3
+    IMG=$4
+ 
+    delete_resource "$RESOURCE_TYPE" "$RESOURCE"
+    create_resource "$RESOURCE_FILE" "$IMG"    
+}
+
```

Comment:
i do love it! just use the boring name and leave the cool one in as a hidden bonus.

me and this friend started making this program while we were in NY and we had all sorts of funky names.. and basically at some point it was just so hard to remember what did what. That really put me off cute names. Maybe it's cool in moderation, but nowadays i always leave the cool name as an alias (for example my todo manager accepts `arrchive` instead of `archive` and it prints skulls and talks to you in pirate speak, but `archive` is still there..)